### PR TITLE
sm64ex: Adapt area_connections slotdata Format

### PR DESCRIPTION
**Pull request by N00byKing**
Required to fix THI behaviour.

New Behaviour: THI Tiny / Huge are treated as seperate paintings. When randomized, this also means that two seperate painting can lead to each THI instance, and the two original THI paintings can lead to different courses

### CODE REVIEW

**Comment by N00byKing** [comment]
Fixing and doing the correct version check tomorrow, don't have time today left sorry :/

File: worlds/sm64ex/__init__.py [hunk 1]
```diff
@@ -54,10 +54,10 @@ def set_rules(self):
         set_rules(self.world, self.player, self.area_connections)
         if self.topology_present:
             # Write area_connections to spoiler log
-            for painting_id, course_id in self.area_connections.items():
+            for painting_id, destination in self.area_connections.items():
                 self.world.spoiler.set_entrance(
-                    sm64courses[painting_id] + " Painting",
-                    sm64courses[course_id],
+                    sm64paintings[painting_id] + " Painting",
+                    sm64courses[int(destination/10)],
```
Comment (Berserker66) [inline worlds/sm64ex/__init__.py:60]: "Integer Division" is // in python, for example:
24 // 10 -> 2

Comment (N00byKing) [reply worlds/sm64ex/__init__.py]: Thats a cool trick! Amended

**Comment by N00byKing** [comment]
Versioning is now (correctly) implemented on both server and client side. Current Version is 0.3.0 because previously it was using 0.2.6, and this is a breaking change.

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**